mlibrary/search
# [LIBSEARCH-146] Update experts panel with link to Experts page

# Overview
Figma designs have been updated to [show `Find more specialists`](https://www.figma.com/file/4izNI8OilJi1MXsNKiKWhX/Library-Search-Design-System?type=design&node-id=599%3A20646&mode=design&t=lyxbFDIHH6xIWE1y-1) linking to [Find a Specialist](https://www.lib.umich.edu/research-and-scholarship/help-research/find-specialist).

This pull request closes [LIBSEARCH-146](https://mlit.atlassian.net/browse/LIBSEARCH-146).

## Anything else?
Code and styling have been simplified for all related components.

### `ShowAllSpecialists`
`ShowAllChildren` was only ever used in `SpecialistsList`, so the component was renamed to `ShowAllSpecialists` and moved to `src/modules/specialists/components/`.

### `SpecialistsWrapper `
`SpecialistsWrapper` was an over engineered component, simply replaced by the `SpecialistList` component. The original component has been deleted.

### Functional Components
To align with [current React practices](https://react.dev/learn/your-first-component), these class components have been converted to functional components:
- `ShowAllSpecialists`
- `RecordList`

## Testing
- Make sure the PR is consistent in these browsers:
- Run accessibility tests:
- Make a [search that includes specialists](http://localhost:3000/everything?query=Asia).
  - Does the `Find more specialists` link appear?
  - Do you see it on the first page of each datastore result?
  - Does the `Show all # Library Specialists` button appear?
    - Does it show the remaining specialists when clicked on?
- Check the rest of the site and see if everything looks okay.
